### DISCUSSION TITLE
CatchAll routes

### DISCUSSION THREAD

**Discussion by jamalsoueidan** [remix-run/remix#9594] [Q&A]
Hello,

I created $ . tsx to catch all, but when I use it with $ . _index and $ . about_us, the outlet doesn't automatically render those pages when the route change.

/dynamic-route/
/dynamic-route/about_us

How to accomplish this? Thank you

**Comment by kiliman** [comment]
The catch-all or splat route matches to the end of the URL. It doesn't support partial prefix. 

You would have to parse the param and render the appropriate route.
